next.config.js

```diff
@@ -1,9 +1,3 @@
-if (process.env.NODE_ENV !== 'production') {
-  require('dotenv').config()
-}
-
-const nextConfig = {
-  env: {}
-}
-
-module.exports = nextConfig
+module.exports = {
```

Comment:
We should maintain nextjs things here

Reply:
Since the release of Next.js 9.4 we now have a more intuitive and ergonomic experience for [adding environment variables](https://nextjs.org/docs/basic-features/environment-variables)